src/javascript/function-scanner.ts

```diff
@@ -118,12 +115,14 @@ class FunctionVisitor implements Visitor {
   }
 
   private _initFunction(
-      node: babel.Node, analyzedName?: string, _fn?: babel.Function) {
-    const comment = getAttachedComment(node);
-    if (!comment) {
-      return;
-    }
-    const docs = jsdoc.parseJsdoc(comment);
+      node: babel.Function|babel.ObjectProperty,
+      analyzedName?: string,
+      docNode?: babel.Node,
```

Comment:
hm, not sure if I like the way clang-format formats a trailing comma in a parameter list

Reply:
Done.